server/controllers/mailing.js

```diff
@@ -34,53 +34,58 @@ class Mailing {
         name: 'Coolest Projects Support',
       },
       subject: 'Welcome on CP',
-      substitutions: {
-        verificationSentence: event.requiresApproval ? 'You will be contacted by the Coolest Projects team if your project is accepted.' : '',
+      dynamic_template_data: {
         eventName: event.name,
         eventDate: event.date,
         eventLocation: event.location,
         eventWebsite: event.homepage,
         eventManageLink: process.env.HOSTNAME,
         projectName: htmlEntities.encode(project.name),
+        ...Mailing.customisationValues(event),
       },
       categories: this.categories.concat([`cp-${event.slug}-registration`]),
-      template_id: '6d20e65f-ae16-4b25-a17f-66d0398f474f',
+      template_id: 'd-23da4e90859043bf81d7c3c1d4c14a5c',
```

Comment:
Seeing as we're using env variables here I wonder if its worth switching this to one.

Reply:
It could have if we'd use different templates_id per instance, but that's the opposite. We're using a single template with customisation per instance instead (we identify the instance in the email via the slug sent via the `customisationValues` function ).